corehq/apps/accounting/mixins.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from datetime import date
+
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.accounting.tasks import (
+    get_unpaid_invoices_over_threshold_by_domain,
+    is_subscription_eligible_for_downgrade_process,
+)
+from corehq.util.quickcache import quickcache
+
+
+@quickcache(['domain_name'], timeout=60 * 60)
+def get_overdue_invoice(domain_name):
```

Comment:
i believe the convention we are trying to follow is use `domain` for the name of the domain and `project`/`domain_obj` for domain object. So we should rename this to `domain`.

Reply:
Is this documented anywhere?

Reply:
Based on book club discussion a couple weeks back I did a bunch of PRs to move towards `domain`/`domain_obj` (https://github.com/dimagi/commcare-hq/pull/23189 - https://github.com/dimagi/commcare-hq/pull/23196)...that said, I left the accounting code alone because it uses `domain_name`/`domain` and it's internally consistent. It also isn't written down for that reason, because there are significant chunks of code that use `domain_name`/`domain`.

`project` is rarely used (even though one of those is `request.project`) and shouldn't be used more.

Reply:
I am not sure if this is documented. It has been raised a couple of times on slack channels.

Reply:
Documented! https://github.com/dimagi/commcare-hq/pull/23809